docs/build/build-win.md

```diff
@@ -151,7 +151,7 @@ CMake will be able to find openssl given the following option is provided:
 ##### 2.1.1.2. Using Installer (Windows)
 
 The 64-bit OpenSSL package for Windows can be downloaded using
-the following link: [Win64OpenSSL_Light-1_1_1c](http://slproweb.com/download/Win64OpenSSL_Light-1_1_1c.exe).
+the following link: [Win32OpenSSL-3_0_5](https://slproweb.com/download/Win32OpenSSL-3_0_5.exe).
```

Comment:
> NOTE: I inserted the link you sent on Slack, but this seems to download the 32-bit version of the installer. Should the correct link be: http://slproweb.com/download/Win64OpenSSL_Light-3_0_5.exe ? If so, let me know and I will make another pull request.

Thanks for noticing! Then probably this way should make sense (not sure about "/" here):

```suggestion
the following link: [Win32OpenSSL-3_0_5](https://slproweb.com/download/Win32OpenSSL-3_0_5.exe) / [Win64OpenSSL-3_0_5](https://slproweb.com/download/Win64OpenSSL-3_0_5.exe).
```